public/src/components/channelManagement/bannerTests/utils/defaults.ts

```diff
@@ -30,6 +33,7 @@ const PROD_DEFAULT_VARIANT: BannerVariant = {
   template: BannerTemplate.ContributionsBanner,
   bannerContent: {
     messageText: '',
```

Comment:
`messageText` needs to be removed from the defaults now

Reply:
Done!